dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolNotResolvable.java

```diff
@@ -176,4 +177,31 @@ public String toString() {
      */
     SYMBOL_NOT_FOUND
   }
+
+  static <U extends SymbolReference> SymbolNotResolvable<U> fromSymbolProblem(
```

Comment:
Can this be an instance method on SymbolProblem so you don't need the switch at all?

Reply:
The switch statement will remain even if I move the method to an instance method on SymbolProblem.
 (Or I’m missing something?)

Reply:
Now the function has no switch statement.